UM/PluginRegistry.py

```diff
@@ -678,6 +688,23 @@ def _locatePlugin(self, plugin_id: str, folder: str) -> Optional[str]:
 
         return None
 
+    def _handleCentralStorage(self, file_data: str, plugin_path: str) -> None:
+        """
+        Plugins can indicate that they want certain things to be stored in a central location.
+        In the case of a signed plugin, this is done by means of the central_storage.json file.
```

Comment:
It's also done this way in the case of a non-signed plug-in, right?

Reply:
You don't _have_ to do it via the central_storage.json file in that case, but yeah. I'll update the documentation